packages/@tailwindcss-postcss/src/__snapshots__/index.test.ts.snap

```diff
@@ -421,6 +421,7 @@ exports[`\`@import 'tailwindcss'\` is replaced with the generated CSS 1`] = `
     color: inherit;
     -webkit-text-decoration: inherit;
     -webkit-text-decoration: inherit;
+    -webkit-text-decoration: inherit;
```

Comment:
🤔

Reply:
> Note: in the diff output you'll notice that some properties are duplicated. These need some fixes in Lightning CSS itself but they don't break anything for us right now.

Reply:
You'll also notice that we already had 2 instances of `-webkit-text-decoration` above it. It's a Lightning CSS bug, or a missing feature depending on how you interpret it.

Reply:
I see, I only looked at you `-webkit-backdrop-filter` fix and thought this might be unrelated :(

Reply:
Yeah, wanted to focus on some other tasks before trying to debug what's happening in Lightning CSS itself. Since this issue already existed this became less of a priority.